.circleci/deploy_testing.sh

```diff
@@ -6,27 +6,20 @@ AWS_APPLICATION_NAME=$2
 export AWS_ENVIRONMENT_NAME=$3
 STATIC_BUCKET=$4
 
-echo 'Deleting configuration files that do not apply to testing'
-rm .ebextensions/03_papertrail.config
+echo 'Deleting configuration files that do not apply to feature deployment'
+rm .ebextensions/04_newrelic.config
```

Comment:
Note here - I noticed we have log aggregation for the feature environment so I kept papertrail. We do not have newrelic, though.